salt/utils/jinja.py

```diff
@@ -98,28 +98,60 @@ def check_cache(self, template):
             self.cached.append(template)
 
     def get_source(self, environment, template):
-        # checks for relative '..' paths
-        if '..' in template:
-            log.warning(
-                'Discarded template path \'%s\', relative paths are '
-                'prohibited', template
-            )
-            raise TemplateNotFound(template)
+        '''
+        Salt-specific loader to find imported jinja files.
+
+        Jinja imports will be interpreted as originating from the top
```

Comment:
This import behavior needs to be in the documentation.  Where is the best location for it?